modules/nw-osp-configuring-external-load-balancer.adoc

```diff
@@ -32,77 +32,154 @@ on {rh-openstack-first}
+* Target all IP addresses on a subnet. This configuration can reduce maintenance overhead,because you can create and destroy nodes within those networks without reconfiguring the load balancer targets. If you deploy your ingress pods by using a machine set on a smaller network, such as a `/27` or `/28`, you can simplify your load balancer targets.
++
+[TIP]
 ====
-You do not need to specify API and Ingress static addresses for your installation program. If you choose this configuration, you must take additional actions to define network targets that accept an IP address from each referenced vSphere subnet.
+You can list all IP addresses that exist in a network by checking the machine config pool's resources.
 ====
 
-.Prerequisites
+.Considerations
 
-* On your load balancer, TCP over ports 6443, 443, and 80 must be reachable by all users of your system that are located outside the cluster.
+* For a front-end IP address, you could use the same IP address for the front-end IP address, the Ingress Controller's load balancer and API load balancer. Check the vendor's documentation for this capability.
 
-* Load balance the application ports, 443 and 80, between all the compute nodes.
+* For a backend IP address, ensure that an IP address for an {product-title} control plane node does not change during the lifetime of the external load balancer. You can achieve by completing one of the following actions:
+** Assign a static IP address to each control plane node.
+** Configure each node to receive the same IP address from the DHCP every time the node requests a DHCP lease. Depending on the vendor, the DHCP lease might be in the form of an IP reservation or a static DHCP assignment. 
 
-* Load balance the API port, 6443, between each of the control plane nodes.
+* For the Ingress Controller backend service, ensure you manually define each node that runs the Ingress Controller in the external load balancer. For example, if the Ingress Controller moves to an undefined node, an connection outage can occur. 
 
-* On your load balancer, port 22623, which is used to serve ignition startup configurations to nodes, is not exposed outside of the cluster.
+.OpenShift API prerequisites
 
-* Your load balancer can access the required ports on each node in your cluster. You can ensure this level of access by completing the following actions:
-** The API load balancer can access ports 22623 and 6443 on the control plane nodes.
-** The ingress load balancer can access ports 443 and 80 on the nodes where the ingress pods are located.
+* You defined a front-end IP address.
+* You ensured that on your load balancer, TCP ports 6443, 443, 22623, and 80 are exposed on the front-end IP address. Check the following items:
```

Comment:
For the api/machine config API, we can remove 443 and 80.  We only need those in the ingress controller section.

Reply:
Thanks. Updated!